src/Pages/Wishlist-page/wishlist.css

```diff
@@ -0,0 +1,34 @@
+.wishlist-page {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 2.5rem minmax(calc(100vh - 7.5rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
```

Comment:
I like the way you are using grid template area.

Reply:
Thankyou